app/assets/javascripts/test/snapshots/public/test-bundle/test/backend-snapshot-tests/timetracking.e2e.js.md

```diff
@@ -31,8 +31,270 @@ Generated by [AVA](https://ava.li).
 
 ## timetracking-timeTrackingForUser
 
-    []
```

Comment:
Nice fix :+1:

Reply:
Can you add a `t.true(timeTrackingEntries.length > 0)` to the tests so that this cannot happen again?